README.md

```diff
@@ -10,25 +10,166 @@
 
 [Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
 
-This is a command line tool (CLI) that help you backup your published Content Model, Content and Assets or move them to a new Contentful space. _It will support Editor Interfaces, Webhooks and Roles & Permissions in a future version._
+This is a library that help you backup your Content Model, Content and Assets or move them to a new Contentful space. _It will support Roles & Permissions in a future version._
 
 To import your exported data, please refer to the [contentful-import](https://github.com/contentful/contentful-import) repository.
 
-## Installation
+## :exclamation: Usage as CLI
+> We moved the CLI version of this tool into our [Contentful CLI](https://github.com/contentful/contentful-cli). This allows our users to use and install only one single CLI tool to get the full Contentful experience.
+>
+> Please have a look at the [Contentful CLI export command documentation](https://github.com/contentful/contentful-cli/tree/master/docs/space/export) to learn more about how to use this as command line tool.
 
-We recommend the installation of this CLI via npm:
+
+## :cloud: Installation
 
 ```bash
-npm install -g contentful-export
+npm install contentful-export
+```
+
+## :hand: Usage
+
+
+While this tool is intended for use as a command line tool, you can also use it as a Node library:
```

Comment:
we should remove the part `While this tool is intended for use as a command line tool`

Reply:
ooops